cmd/k8s-operator/proxygroup_specs.go

```diff
@@ -80,6 +81,13 @@ func pgStatefulSet(pg *tsapi.ProxyGroup, namespace, image, tsFirewallMode, cfgHa
 									})
 								}
 
+								if pg.Spec.Type == tsapi.ProxyGroupTypeEgress {
+									mounts = append(mounts, corev1.VolumeMount{
+										Name:      fmt.Sprintf(egressSvcsCMNameTemplate, pg.Name),
```

Comment:
I know it's trivial, but as this is repeated quite a few times, it would be nice to make a small `pgEgressConfigMapName` function for it (no strong opinions on the exact name ofc)